### PR TITLE
Possible fix for binoc perma zoom

### PR DESCRIPTION
**Pull request by morrowwolf**
# About the pull request

I have no consistent reproducibility for this but this may fix it? Seems to occur when an explosion blows up the binocs in the hand of the person while zoomed.

# Explain why it's good for the game

Bug bad

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Morrow
fix: Possible fix for binoc perma zoom
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
i think this would fixes this issue : 
https://github.com/cmss13-devs/cmss13/issues/3405
can you confirm?

**Comment by morrowwolf** [comment]
> i think this would fixes this issue : #3405 can you confirm?

Not 100% it'll fix it so we'll leave it for now

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -829,6 +829,7 @@ cases. Override_icon_state should be a list.*/
 	UnregisterSignal(src, list(
 		COMSIG_ITEM_DROPPED,
 		COMSIG_ITEM_UNWIELD,
+		COMSIG_PARENT_PREQDELETED,
```
Comment (harryob) [inline code/game/objects/items.dm:832]: use COMSIG_PARENT_QDELETING

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -861,6 +862,7 @@ cases. Override_icon_state should be a list.*/
 		RegisterSignal(src, list(
 			COMSIG_ITEM_DROPPED,
 			COMSIG_ITEM_UNWIELD,
+			COMSIG_PARENT_PREQDELETED,
```
Comment (harryob) [inline code/game/objects/items.dm:865]: ditto

**Pull request merged**